vtex-apps/order-quote
# Feature/recover custom data

#### What does this PR do? \*
Saves and Recover customData values in the quote

#### How to test it? \*
- Set customFields to the checkout in you account
- Add products to the cart
- Set customData with your customFields
- Head over to /orderquote/create to create one quotation (check the option "Clear cart after saving")
- Back on listing page /orderquote, select your quotation, click "Use"
- Check if your orderForm has the customData properties


#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*
